app/Actions/VerifyDependencies.php

```diff
@@ -30,6 +30,19 @@ class VerifyDependencies
         ],
     ];
 
+    private $optionalDependencies = [
+        [
+            'command' => 'gh',
+            'label' => 'Unofficial GitHub command line tool',
+            'instructions_url' => 'https://github.com/github/hub',
+        ],
+        [
+            'command' => 'hub',
+            'label' => 'Official GitHub command line tool',
+            'instructions_url' => 'https://cli.github.com/',
+        ],
```

Comment:
I think you have the `command` lines mixed up between these two

Reply:
Geez 🤦‍♂️